src/home/home.js

```diff
@@ -11,6 +12,11 @@ const initializeHome = () => {
         pokemon.id = index + 1;
         pokemonContainer.insertAdjacentHTML('beforeend', pokemonCard(pokemon));
       });
+      const btnModal = document.querySelectorAll('button');
+      console.log(btnModal);
```

Comment:
If this console.log is not needed delete it

Reply:
Sorry for that. I already deleted it.